Views/TweetCell.m

```diff
@@ -21,4 +24,107 @@ - (void)setSelected:(BOOL)selected animated:(BOOL)animated {
     // Configure the view for the selected state
 }
 
+- (IBAction)didTapFavorite:(id)sender {
+    NSLog(@"tapped favorite");
+    if(self.tweet.favorited)
```

Comment:
nit: spacing

Reply:
What does favorited mean for a tweet? I'm concerned reading this because it doesn't check anything about the favorite count here but it does decrement it. Do you need to worry about negative numbers?